Source/WebGPU/WGSL/GlobalVariableRewriter.cpp

```diff
@@ -319,6 +319,134 @@ void RewriteGlobalVariables::visitEntryPoint(AST::Function& function, AST::Stage
     insertLocalDefinitions(function, usedGlobals.privateGlobals);
 }
 
+static BindGroupLayoutEntry::BindingMember bindingMemberForGlobal(auto& global)
```

Comment:
@tadeuzagallo @djg  no idea if this is correct.

Seems to work on the GitHub chrome samples